blueprints/ember-frost-test/index.js

```diff
@@ -2,16 +2,26 @@
  * Install blueprint for ember-frost-test addon
  */
 
+var Promise = require('bluebird')
```

Comment:
I didn't think native promises had a `promisify` or `denodify` helper to convert callback based node methods to promised based methods.